payload-schemas/api.github.com/branch_protection_rule/edited.schema.json

```diff
@@ -14,16 +14,24 @@
         "admin_enforced": {
           "type": "object",
           "required": ["from"],
-          "properties": { "from": { "type": "boolean" } },
+          "properties": {
+            "from": {
+              "$ref": "common/branch-protection-rule-boolean.schema.json"
+            }
+          },
```

Comment:
I see the benefit of moving the `enum` in `branch-protection-rule-enformcement-level.schema.json` to centralize the possible values in one place and maintainability.

What is the benefit of doing the same for `integer`, `boolean` and `string[]` here? For sure there is a reason but I'm not understanding it.

Reply:
There isn't really that much of a benefit. It's is inline with `octokit/webhooks.net` though